### ISSUE TITLE
Trouble Applying the 1.6.7  version

### ISSUE THREAD

**Issue by AlexanderMadani** [1gravity/Android-RTEditor#153]
Hi everyone,  I am the developer of  EZ Notes, and I use this Android-RTEditor technology for my Editor.

For the longest time, I have been on version 1.6.1  (and it worked fine).  But recently, when I published my App to my device Nexus 6 device running API 25  (Android 7.1.1  -- Nougat), I noticed that the Camera function would not launch the camera.  It would only take me to a blank page (gray in color) and not do anything and I would have to back out of that.

Bear in mind that my App supports minSdkVersion 14  all they way up
and I am targetting TargetSdkVersion 27

So i decided to give the new version of this RTEditor library  (ver. 1.6.7)  a try.   The Sync process went okay, but during the Compile (build process) Android Studio informed me that I needed to set my minSdkVersion = 26  or above !   I was rather surprised and a bit dismayed since I need to support 14 and up.

But I went ahead and decided to set my minSdkVersion = 26  (and test on an emulator).  The problem then occurs where after I enter some data into my Editor and try to Save my Edit Session, my App crashes.  This was not occuring with version 1.6.1  of your library, and I have no idea what is causing it.

I hope someone can so kindly reach me and let me know what I need to do to resolve these issues. My project has been coded carefully and per your ReadMe documentation.  So ideally, I should have just been able to change the dependency library version from 1.6.1  to 1.6.7  and have it begin providing the latest enhancements.  But that is not the case.

To reach me, my email is:   amadani@pristineusa.com
I am also on G+ under Alex M, and my page is at:  ➙ http://tinyurl.com/kowp325

Thank you to anyone who will help my customers and I on this issue.

Sincerely Yours
Alexander Madani
Developer of:  EZ Notes

**Comment by ohenepee** [comment]
I am having a similar issue... I think it has to do with the **ext.kotlin_version** in this library. It has to be updated from `1.2.21` to `1.2.30`. @1gravity 

Android Studio fails to build when I add this library and I also keep getting an __outdated Kotlin runtime warning__ and it warns that the Android Studio (latest stable not beta nor canary) is using the kotlin plugin version 1.2.30 therefore I need to update from 1.2.21 to 1.2.30

**Comment by ohenepee** [comment]
@AlexanderMadani did u solve the issue or you went back to use the old version which you were initially using?

**Comment by AlexanderMadani** [comment]
Hello, 

I ended up using the latest version 1.6.7  , and it seems to be working okay.

The only minor issue that I hope the project developers will perhaps kindly address in the next version (version 1.6.8  I guess)  is the following.

I have observed that in order for the Camera (Photo Capture) feature to work, that the user must ensure the App has  “Permission to Storage”  

Unfortunately, when the user goes into the Editor right after installing my App, and tries to use the Photo Capture button, there is no Dialog Prompt that will pop-up to request this Permission to Storage.

Ideally, there should be, so that the user grants permission from within the App  (rather than from outside the App)... and can begin using the Photo Capture function right away.


But aside from this, I can say that it appears that version 1.6.7  has a Working Photo Capture function  (at least from all the people I asked thus-far).


Thank you so much for following up on this with me.


Kind Regards,
Alexander Madani
about.me/alexander.madani



Developer of:  EZ Notes
Available on Google Play for Android 
Download at:  App link
